client.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// entry code goes here
+var Irken = require('irken');
+
+var app = new Irken();
+
+var plugins = [
+  {
+    register: require('holovisor')
+  },
+  {
+    register: require('./plugins/appbar'),
+    options: {
+      title: 'Parallax Chrome IDE'
+    }
+  },
+  {
+    register: require('./plugins/sidebar')
+  },
+  {
+    register: require('./plugins/editor'),
+    options: {
+      initial: `function helloWorld(hello){\n  hello = 'world';\n}`
+    }
+  }
+];
+
+app.register(plugins, function(err){
+  console.log('registered', err, app);
```

Comment:
just as a general rule its better to have a logging system in place that can be tuned for different levels of output.  This one is probably helpful this early in development :)